CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # `dev-master`
 
+* [#578](https://github.com/atoum/atoum/pull/578) Fix arguments priority parsing when they have no priority ([@agallou])
```

Comment:
Please, can you add this to a Bugfix section (example here: https://github.com/atoum/atoum/blob/master/CHANGELOG.md#bugfix)